xml/schema/CRM/Multicompanyaccounting/Company.entityType.php

```diff
@@ -0,0 +1,10 @@
+<?php
+// This file declares a new entity type. For more details, see "hook_civicrm_entityTypes" at:
+// https://docs.civicrm.org/dev/en/latest/hooks/hook_civicrm_entityTypes
+return [
+  [
+    'name' => 'Company',
+    'class' => 'CRM_Multicompanyaccounting_DAO_Company',
+    'table' => 'civicrm_company',
```

Comment:
Same here, the table name in other places is `multicompanyaccounting_company`

Reply:
fixed